test/fixtures/circular-example.yaml

```diff
@@ -3,22 +3,23 @@ info:
   version: 1.0.0
   title: Test circular reference in example
 host: petstore.swagger.io
+produces:
+  - application/json
 paths:
   /:
     get:
       responses:
         200:
           description: Get company info
-          examples:
-            application/json:
-              example:
-                $ref: '#/definitions/Company'
```

Comment:
This wasn't valid example, looking at Swagger spec "Reference Object" is not listed inside "Example Object".

https://swagger.io/specification/v2/#exampleObject